src/common/options/mds.yaml.in

```diff
@@ -1421,3 +1421,13 @@ options:
   default: false
   flags:
   - runtime
+- name: mds_symlink_recovery
```

Comment:
Does this needs to be a config option? IOW, what's the need to keep it disabled? (since without these changes,  symlink is recovered as a regular file - which is a bug!)

Reply:
I'm on the fence about it. I think it could be turned on by default.

Reply:
Ack, I will turn it on by default. We can anyway turn in it off if it's affecting performance in anyway.